sql/item_sum.cc

```diff
@@ -3553,6 +3553,57 @@ String *Item_sum_udf_str::val_str(String *str)
 #endif /* HAVE_DLOPEN */
 
 
+Item_sum_str::Item_sum_str(THD *thd)
+  : Item_sum(thd)
+{
+}
+
+Item_sum_str::Item_sum_str(THD *thd, Item_sum_str *item)
+  : Item_sum(thd, item)
+{
+}
+
+Item_sum_str::Item_sum_str(THD *thd, Item *item_par)
+  : Item_sum(thd, item_par)
+{
+}
+
```

Comment:
The implementations of the constructors above could probably go directly to item_sum.h